README.md

```diff
@@ -249,9 +249,9 @@ As an example we use the Symfony `DateTimeNormalizer` service so we do have supp
 ```yaml
     # Add DateTime Normalizer to Dunglas' Doctrine JSON ODM Bundle
     dunglas_doctrine_json_odm.serializer:
-        class: Symfony\Component\Serializer\Serializer
```

Comment:
I ran into the problem with `autoconfigure` feature here (which is enabled by default).
In default environment `dunglas_doctrine_json_odm.serializer` auto-tags as `normalizer`. This leads to the situation when default symfony's serializer uses `dunglas_doctrine_json_odm.serializer` instead of `objectNormalizer`. As a result you cant instantiate new objects through standard symfony serializer anymore.
To solve this just use `autoconfigure: false`.